src/main/java/org/dasein/cloud/ci/ConvergedHttpLoadBalancer.java

```diff
@@ -30,42 +30,42 @@
     private List<BackendServiceBackend> backendServiceBackends = new ArrayList<BackendServiceBackend>();
```

Comment:
Since you are updating this class, might you endeavour to provide the annotations too?